Documentation/contributing/development/contributing_guide.rst

```diff
@@ -127,36 +143,48 @@ requirements have been met:
    | ``kind/bug``                 | This is a bugfix worth mentioning in the release notes                    |
    +------------------------------+---------------------------------------------------------------------------+
-   | ``kind/enhancement``         | This is an enhancement/feature                                            |
+   | ``kind/enhancement``         | This enhances existing functionality in Cilium                            |
    +------------------------------+---------------------------------------------------------------------------+
-   | ``priority/release-blocker`` | This PR should block the current release                                  |
+   | ``kind/feature``             | This is an feature                                                        |
    +------------------------------+---------------------------------------------------------------------------+
-   | ``area/*``                   | Code area this PR covers                                                  |
+   | ``priority/release-blocker`` | This PR should block the current release                                  |
    +------------------------------+---------------------------------------------------------------------------+
    | ``needs-backport/X.Y``       | PR needs to be backported to these stable releases                        |
    +------------------------------+---------------------------------------------------------------------------+
-   | ``pending-review``           | PR is immediately ready for review                                        |
-   +------------------------------+---------------------------------------------------------------------------+
-   | ``wip``                      | PR is still work in progress, signals reviewers to hold.                  |
-   +------------------------------+---------------------------------------------------------------------------+
    | ``backport/X.Y``             | This is backport PR, may only be set as part of :ref:`backport_process`   |
    +------------------------------+---------------------------------------------------------------------------+
    | ``upgrade-impact``           | The code changes have a potential upgrade impact                          |
    +------------------------------+---------------------------------------------------------------------------+
+   | ``area/*`` (Optional)        | Code area this PR covers                                                  |
+   +------------------------------+---------------------------------------------------------------------------+
 
    .. note:
 
       If you do not have permissions to set labels on your pull request. Leave
       a comment and a core team member will add the labels for you. Most
       reviewers will do this automatically without prior request.
 
+#. GitHub provides the ability to create a Pull Request in "draft" mode.
```

Comment:
Might be worth starting this paragraph with a short sentence "Open a draft pull request." then proceed with the explanation of how and why.